src/test/e2e/pageobjects/Home.page.ts

```diff
@@ -12,9 +12,16 @@ export class HomePage extends CommonPage {
     return $(helpers.MainHeader).getText();
   }
 
+  // Old function for previous start page screen - leaving in case of future reversion.
   async clickStartNowButton(): Promise<ViewOptionPage> {
     const button = await $(helpers.StartNowButton);
     button.click();
     return new ViewOptionPage();
   }
```

Comment:
yeet it outta there, if we need it we can revert

Reply:
sorted